File: Java/Iniciante/1050 - DDD.java
import java.util.Scanner;

/*
 * Autor: Pierre Vieira
 * Data de submissão: 09/02/2020 18:18:26
 */
public class Main {

    public static void main(String[] args) {
        int ddd;
        Scanner teclado = new Scanner(System.in);
        ddd = teclado.nextInt();
        switch (ddd){
            case 61:
                System.out.println("Brasilia");
                break;
            case 71:
                System.out.println("Salvador");
                break;
            case 11:
                System.out.println("Sao Paulo");
                break;
            case 21:
                System.out.println("Rio de Janeiro");
                break;
            case 32:
                System.out.println("Juiz de Fora");
                break;
            case 19:
                System.out.println("Campinas");
                break;
            case 27:
                System.out.println("Vitoria");
                break;
            case 31:
                System.out.println("Belo Horizonte");
                break;
            default:
                System.out.println("DDD nao cadastrado");
                break;
        }
    }
}

File: C/Iniciante/1050 - DDD.c
/*
 * Autor: Pierre Vieira
 * Data de submissão: 09/02/2020 18:14:11
 */
#include <stdio.h>
int main(){
    int ddd;
    scanf("%d", &ddd);
    switch (ddd){
        case 61:
            printf("Brasilia");
            break;
        case 71:
            printf("Salvador");
            break;
        case 11:
            printf("Sao Paulo");
            break;
        case 21:
            printf("Rio de Janeiro");
            break;
        case 32:
            printf("Juiz de Fora");
            break;
        case 19:
            printf("Campinas");
            break;
        case 27:
            printf("Vitoria");
            break;
        case 31:
            printf("Belo Horizonte");
            break;
        default:
            printf("DDD nao cadastrado");
            break;
    }
    printf("\n");
    return 0;
}